### DISCUSSION TITLE
Request for Nova Theme Support in PrimeNG 18

### DISCUSSION THREAD

**Discussion by ashishmondal** [primefaces/community#2954] [PrimeNG]
Hello PrimeNG Support Team,

We’re currently using PrimeNG 16 with the Nova theme, which aligns well with our design requirements. As we’re planning to upgrade our application to PrimeNG 18, we noticed that the Nova theme is not supported with this version due to the new theming architecture introduced. Given this, we would like to inquire:

1. Is there any plan to reintroduce or support the Nova theme in PrimeNG 18?
2. If not, are there specific recommended steps to replicate Nova’s look and feel using the available themes?

Our team relies on Nova’s design for consistency across our application and would appreciate any guidance on how to proceed.

Thank you for your time and support!

**Comment by qwadrox** [comment]
@ashishmondal Hi.

> Premium Templates
> Application templates like Apollo do not support PrimeNG v18 yet, all templates will be updated to PrimeNG v18 by the end of the 2024.

https://v18.primeng.org/guides/migration#breaking

**Reply by klanglie** [reply]
> @qwadrox 
> 
> > Premium Templates
> > Application templates like Apollo do not support PrimeNG v18 yet, all templates will be updated to PrimeNG v18 by the end of the 2024.
> 
> https://v18.primeng.org/guides/migration#breaking

Is this still in the works? I downloaded my Extended themes which include Babylon v16, Omega v16, California v17 and Roma v17. It would be very helpful if these could all be upgraded to PrimeNG v19 and Angular 19 real soon. Thanks!

**Reply by qwadrox** [reply]
@cagataycivici

**Comment by DanielOberlechner** [comment]
@qwadrox 
Hi,
I dont know if you are a maintainer of the primeng project, but you people have a lot of broken links like yours here on the comment and on google as well. Maybe take some time to make the right adjustments to fix the broken links.

In this case the correct link would be:
https://primeng.org/guides/migration#breaking

Kind regards,
Daniel

**Comment by JimTudeski** [comment]
but here isn't nova theme... :(

**Comment by binayag1** [comment]
Hi Any update on this one?

**Comment by JimTudeski** [comment]
18/19 version is still terrible, we need to stay on v17...

**Comment by roman-bychkov** [comment]
I'm waiting for PrimeNG's unstyled mode, in which I can apply my old CSS styling and override this new design token nightmare.

**Reply by JimTudeski** [reply]
agree

**Comment by ctodesco** [comment]
What a bafflingly idiotic decision.  Those of us using PrimeNG in our apps can't have it suddenly change appearance because we're updating Angular.  So now I have to do a ton of work just to restore it to what it looked like before-- you couldn't have included a "legacy" theme that looked the same as Nova?

**Comment by cagataycivici** [comment]
We've recreated Lara and Material Themes as they were the most commonly used on top of the new Aura. Recreating all of them would be a huge maintenance work for us so the legacy ones like Nova were left out.
